### ISSUE TITLE
Group - Selected members are not highlighted

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#40429]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v1.4.63-0
**Reproducible in staging?:** Y
**Reproducible in production?:** No, new feature
**Email or phone of affected tester (no customers):** natnael.expensify+3@gmail.com
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Click on ‘FAB’ > Start chat
2. Select multiple users > Click on ‘Next’ button

## Expected Result:
Selected users should be highlighted

## Actual Result:
Selected user are not highlighted and members list UI is different from production. It looks like the older version of the page.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/e6185eb5-6019-4638-bce3-40b037905af1

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by izarutskaya** [comment]
@joekaufmanexpensify I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by izarutskaya** [comment]
We think this issue might be related to the #vip-vsb.

**Comment by izarutskaya** [comment]
Production 


https://github.com/Expensify/App/assets/115492554/392dc0b7-87ba-41d5-b856-e404b9e5dfbd

**Comment by MahmudjonToraqulov** [comment]
## Proposal
### Please re-state the problem that we are trying to solve in this issue.
Group - Selected members are not highlighted.

### What is the root cause of that problem?
Some stylea are wrong.


### What changes do you think we should make in order to solve the problem? 
 We should add " `pressableStyle={[[styles.selectionListPressableItemWrapper, item.isSelected && styles.activeComponentBG, isFocused && styles.sidebarLinkActive]]}`" here:
https://github.com/Expensify/App/blob/9b839f4f18d724dba1ed17ed5967f7e033781428/src/components/SelectionList/InviteMemberListItem.tsx#L51

And "`containerStyle={styles.mb3}`" here:
https://github.com/Expensify/App/blob/9b839f4f18d724dba1ed17ed5967f7e033781428/src/components/SelectionList/InviteMemberListItem.tsx#L53

Also remove "`styles.optionRow`" here:
https://github.com/Expensify/App/blob/9b839f4f18d724dba1ed17ed5967f7e033781428/src/components/SelectionList/InviteMemberListItem.tsx#L95


### What alternative solutions did you explore? (Optional)
N/A


### Result

**Comment by joekaufmanexpensify** [comment]
I don't think this is a bug. Here is the mockup from the design doc which matches the UI shown above on staging. 

<img width="182" alt="image" src="https://github.com/Expensify/App/assets/54866469/8f798724-7ec9-483c-8370-1af5839dfef9">

**Comment by joekaufmanexpensify** [comment]
cc @amyevans, unless you disagree, IMO we should close this, it seems intended.
